src/core/probe/bpf/include/common_defs.h

```diff
@@ -110,4 +110,6 @@ static __always_inline void err_report(u64 sym_addr, u32 pid)
 #define unlikely(x) __builtin_expect(!!(x), 0)
 #endif
 
+#define DIV_CEIL(m, n) (((m) + (n) - 1) / (n))
```

Comment:
I think `m + n` could overflow here; instead `1 + ((m) - 1) / (n)` can be used. Not that it would happen here but if that is reused in other code paths.